test/unit/media.html5.js

```diff
@@ -71,6 +71,19 @@ test('test playbackRate', function() {
   strictEqual(tech.playbackRate(), 0.75);
 });
 
+test('should remove the controls attribute when recreating the element', function() {
```

Comment:
The first test fails because controls are always true on the iPhone (or at least the two I tested). I can add an `if (vjs.IS_IPHONE == false`, but I'd like to know if there's any bigger implications of this.